src/js-form-validator.ts

```diff
@@ -313,10 +313,8 @@ export class FormValidator {
       case 'phone':
         if (!allowEmpty && !value.length) return -1;
         let vals = value.split('x');
-        for (let i = 0, l = vals.length; i < l; i++)
-          vals[i] = vals[i].replace(/\D/g, '');
         value = vals.join('x').replace(/^1/, ''); // remove leading 1 if added, only for US
-        status = /^[0-9x]{7,16}$/.test(value) ? 1 : 0;
+        status = /^[0-9\-\+\s\(\)]{7,16}$/.test(value) ? 1 : 0;
```

Comment:
@Izzmo here is the example that was able to get past the phone number validation.

https://files.slack.com/files-pri/T04SMMDAY-F5UGL2M09/pasted_image_at_2017_06_16_01_25_pm.png
![image](https://user-images.githubusercontent.com/1868262/27305272-9c1bad1a-5506-11e7-96b7-d454862a0eab.png)